tensorflow/python/ops/math_ops.py

```diff
@@ -2839,7 +2950,7 @@ def matmul(a,
          [139, 154]], dtype=int32)>
 
   A batch matrix multiplication with batch shape [2]:
-
+  
```

Comment:
Trailing whitespace. Please configure your editor/IDE to trim whitespaces on save.